specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/examples/AgentPoolsCreate_Flex.json

```diff
@@ -0,0 +1,155 @@
+{
+  "parameters": {
+    "api-version": "2023-08-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "agentPoolName": "agentpool1",
+    "parameters": {
+      "properties": {
+        "orchestratorVersion": "",
+        "count": 3,
```

Comment:
Can count co-exist w/ flexProfile? I thought no?

Reply:
RP would reject this right?

Reply:
Right RP should reject this - I'll update this example.